src/codeu/chat/client/core/Context.java

```diff
@@ -40,11 +40,17 @@ public UserContext create(String name) {
         null :
         new UserContext(user, view, controller);
   }
-  // Uses the instance variable view which has a getInfo method that returns a serverInfo object 
+  // Uses the instance variable view which has a getInfo method that returns a serverInfo object
   // based off the version of the server that the user is running.
+
   public ServerInfo getInfo() {
     return this.view.getInfo();
   }
+
+  public Info getInfo() {
```

Comment:
I believe that I made the method above and he didnt realize it and tried to make another one